demos/node-classification/attri2vec/attri2vec-citeseer-node-classification-example.ipynb

```diff
@@ -104,6 +104,7 @@
       "\n",
```

Comment:
<p>Comment should be updated to say 20% and 80% if we require 20% to avoid warnings</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stellargraph/stellargraph/pull/1008/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

Reply:
nice catch!